iio.h

```diff
@@ -376,23 +360,22 @@ __api __check_ret struct iio_context * iio_create_network_context(const char *ho
  *   ('<b>\0</b>' none, 'x' Xon Xoff, 'r' RTSCTS, 'd' DTRDSR) parts separated
  *   with a comma. For example <i>"serial:/dev/ttyUSB0,115200"</i>,
  *   <i>"serial:/dev/ttyUSB0,115200,n,8,1"</i>*/
-__api __check_ret struct iio_context * iio_create_context_from_uri(const char *uri);
-
+__api __check_ret struct iio_context *
+iio_create_context_from_uri(const char *uri);
```

Comment:
i wonder if for these, it works to do:
```
__api __check_ret
struct iio_context * iio_create_context_from_uri(const char *uri);
```

if we need to break the line, it may worth putting the specifiers on a separate line

Reply:
I'm wondering - shouldn't this become 
*iio_create_context_from_uri(const char *uri)

"When declaring pointer data or a function that returns a pointer type, the preferred use of * is adjacent to the data name or function name and not adjacent to the type name. Examples:"

Reply:
PointerAlignment can be set to Left, Right or Middle.